### ISSUE TITLE
Game crashes on arenas and festivals after custom song finishes when all four main players are connected (RPCS3)

### ISSUE THREAD

**Issue by ghost** [hmxmilohax/rock-band-3-deluxe#62]
Whenever I try to do anything with all four players connected (bass, drums, guitar, and vox), specifically with anything that could remotely pass as a custom song (actual customs, decensors, 2x bass versions) the game crashes as soon as it starts getting to the results screen (yet it hasn't done so just yet for the RB4 stuff from C0Assassin). In some cases it has even crashed somewhere in the middle. I have tested this with and without autoplay, even making a setlist and still get the same results. It seems to work just fine in the clubs (both small and big) and music videos with that many people and in any venue with 1-3 players (not a full band). As soon as I add that fourth player, pick an arena or festival, and play a custom though (doesn't matter if it's by itself or part of a setlist), it bugs at the results. I have an image attached below with what I keep getting.

![Sad](https://user-images.githubusercontent.com/114825943/193465215-3290e616-ed2a-48c1-8d12-f9706489ce03.PNG)

TL;DR: RB3DX no likey full band customs for big shows

**Comment by jnackmclain** [comment]
I believe this has been fixed with [6cae197](https://github.com/hmxmilohax/rock-band-3-deluxe/commit/6cae1974cef3a57ea7f08f09dbccf4e4f2aec9a4)

Reopen if not
